integration-test/sdk.nfts.transfers.test-integration.ts

```diff
@@ -366,4 +369,49 @@ describe('E2E Test: Sdk (read)', () => {
       );
     });
   });
+
+  describe('As an account I should get list of transfers by a contract address', () => {
+    const contractAddress = '0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d'; //BAYC
+    jest.setTimeout(10000);
```

Comment:
why did you need to put the setTimeout here?

Reply:
because sometimes I notice that Moralis takes quite a lot to respond, so I do not want the test to fail if it's Moralis related and it returns data after all